dttools/src/libforce_halt_enospc.c

```diff
@@ -55,6 +48,21 @@ int open(const char *path, int flags, ...)
 	fd = original_open(path, flags, mode);
 
 	if(fd == -1 && errno == ENOSPC) {
+		int err_fd;
+		char *filename;
+		char *parent = "../";
```

Comment:
In this case, CCTOOLS_DISK_ALLOC is unique for each task if I am remembering environment variables correctly. To address using PID instead of taskid, we get PID once we fork. At that point, I believe it is out of our hands which is why I am using taskid since we know that before we ship out the task.